### ISSUE TITLE
Enhancement - Copy Text

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#21]
```
It would be great if you could select text from the document and copy it 
to your clipboard.
```

Original issue reported on code.google.com by `charles....@gmail.com` on 26 Feb 2010 at 1:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@gmail.com` on 26 Feb 2010 at 1:35
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Let me know if you need any help maintaining or adding enhancements to this 
project.
```

Original comment by `charles....@gmail.com` on 26 Feb 2010 at 3:41

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@gmail.com` on 16 Mar 2010 at 6:57
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I use FlexPaper with students and find the copy-text feature the most needed
enhacement at the moment. As is print control, this should also be an optional 
feature.
```

Original comment by `pkaszub...@gmail.com` on 29 Mar 2010 at 11:28

**Comment by GoogleCodeExporter** [comment]
```
Hi Erik, I'm  also interested in the select/copy feature, do you have a 
prevision for
this.
thank's in advance
```

Original comment by `studio.w...@gmail.com` on 6 Apr 2010 at 2:29

**Comment by GoogleCodeExporter** [comment]
```
will be part of 1.4.0 released end of July
```

Original comment by `erik.eng...@gmail.com` on 7 Jul 2010 at 12:11
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@gmail.com` on 7 Jul 2010 at 12:12
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
available now in 1.4.0
```

Original comment by `erik.eng...@gmail.com` on 25 Jul 2010 at 10:58
- Changed state: **Fixed**
